docs/README.md

```diff
@@ -295,6 +296,16 @@ __Note:__ The default option is `never`.
 
 > :warning: Due to a ["won't fix" bug in the Kubernetes API](https://github.com/kubernetes/kubernetes/issues/41743), the LogDNA agent collects events from the entire cluster, including multiple nodes. To prevent duplicate logs when running multiple pods, the LogDNA agent pods defer responsibilty of capturing events to the oldest pod in the cluster. If that pod is down, the next oldest LogDNA agent pod will take over responsibility and continue from where the previous pod left off.
 
+### Configuring Metrics Reporter Statistics
```

Comment:
Do you think it's worth add a little more information here? Maybe explain how this feature differs from other features similar to it, like the K8's events/enrichment? Is this eventually going to replace the current features?

I could see an argument both ways as adding too much information might be confusing to the client.

Reply:
I don't think you're wrong, as I get confused. I'll add a bit more.